webapp-django/crashstats/crashstats/static/crashstats/js/socorro/correlation.js

```diff
@@ -42,10 +42,7 @@ window.correlations = (function () {
                     'date': totals['date'],
                 };
 
-                var channels = ['release', 'beta', 'aurora', 'nightly'];
-                if (product === 'Firefox') {
-                    channels.push('esr');
-                }
+                var channels = $('#mainbody').data('channels');
```

Comment:
So, before it used to *only* add `esr` to the list **if** the product was `Firefox`. Now `esr` is always in that list. But I don't care. It's just too much hassle to complicate things since I don't think we have any correlations in Telemetry for non-Firefox products.